cognite/pygen/utils/helper.py

```diff
@@ -21,3 +21,35 @@ def chdir(new_dir: Path) -> Iterator[None]:
 
     finally:
         os.chdir(current_working_dir)
+
+
+def _get_literal_str(text: str) -> str | None:
+    """Finds the Literal in a string and returns it.
+
+    Args:
+        text: The string to search for the Literal.
+
+    Returns:
+        The Literal in the string, or None if not found.
+
+    Examples:
+
+        >>> _get_literal_str("Literal['hello']")
+        "Literal['hello']"
+        >>> _get_literal_str("str | int")
+
+        >>> _get_literal_str("Literal['hello', 'world'] | int")
+        "Literal['hello', 'world']"
+        >>> _get_literal_str('None | Literal["hello", "world", "!"]')
+        'Literal["hello", "world", "!"]'
```

Comment:
These unit test are executed.

Reply:
Shit, sorry for that. I was not aware you are using doctests here :). Disregard it